server/db/schema.rb

```diff
@@ -10,15 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_10_18_151259) do
+ActiveRecord::Schema[7.0].define(version: 2022_10_19_162136) do
+  create_table "companies", force: :cascade do |t|
+    t.string "name"
+    t.text "description"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
   create_table "users", force: :cascade do |t|
     t.string "email"
     t.string "password_digest"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "usertype"
     t.boolean "email_confirmed", default: false
     t.string "confirm_token"
-    t.string "usertype", default: "student"
```

Comment:
One quickie: could we have the default value back?

Reply:
My current unmerged PR kinda depends on it.

Reply:
solved